besecure_developer_toolkit/src/vdnc.py

```diff
@@ -53,8 +53,14 @@ def verify_versiondetails_name(self):
         """It checks the version details file naming
         convention that is uploaded by user"""
         obj = OSSPMaster(self.issue_id, self.name)
-        obj.check_issue_exists(self.issue_id)
-        obj.check_repo_exists(self.name)
+        if obj.check_issue_exists(self.issue_id) is False:
```

Comment:
can you change the file name from vdny.py to something meaning full name.

Reply:
@sudhirverma changed.